fastlane-env-reminder/docs/contributors.md

```diff
@@ -0,0 +1,29 @@
+# Contributors
+
+### Build and Test
+
+```bash
+$ npm run build
+$ npm test
+```
+
+### Checkin
+
+- Do checkin source (src)
+- Do checkin build output (lib)
+- Do not checkin runtime node_modules in development workflow (see [Dev Workflow documentation](https://github.com/actions/toolkit/blob/master/docs/javascript-action.md#dev-workflow))
+- Do not checkin devDependency node_modules (husky can help see below)
```

Comment:
```suggestion
- Do not check in devDependency `node_modules` (husky can help, see below)
```

although this might not be necessary with the sentence before